benchmarks/bench.jl

```diff
@@ -0,0 +1,39 @@
+using Libm
+using BenchmarkTools
+
+xerf1 = linspace(-0.84375,0.84375,100)
+xerf2 = linspace(-2e-28,2e-28,100)
+xerf3 = linspace(-0.84375,0.84375,100)
+xerf4 = linspace(0.84375, 6, 100)
+xerf5 = linspace(-6,-0.84375, 100);
+xerf = union(xerf1,xerf2,xerf3,xerf4,xerf5);
+
+xerfc1 = linspace(-0.84375,0.84375,100)
+xerfc2 = linspace(-2e-56,2e-56,100)
+xerfc3 = linspace(-0.25,0.25,100)
+xerfc4 = linspace(0.84375, 28, 100)
+xerfc5 = linspace(-28,-0.84375, 100);
```

Comment:
Perhaps, for benchmarking you need 100x more datapoints, because measuring <100 microseconds intervals is not accurate.